google/auth/transport/aiohttp_req.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2020 Google LLC
```

Comment:
is there a reason to abbreviate the name of this file?

Reply:
Abbreviated due to aiohttp_request appearing lengthy when called by the user (aiohttp_request.AuthorizedSession()) and it seemed relatively clear that req denoted request. Could change this though if the longer version or a better name is more suitable.